hazelcast/src/main/java/com/hazelcast/jet/core/ProcessorMetaSupplier.java

```diff
@@ -186,17 +192,24 @@ default boolean closeIsCooperative() {
      *              Note that it might not be the actual error that caused the job
      *              to fail - it can be several other exceptions. We only guarantee
      *              that it's non-null if the job didn't complete successfully.
+     * @see #isReusable()
      */
     default void close(@Nullable Throwable error) throws Exception {
     }
 
     /**
-     * Returns {@code true} if this instance is reusable, i.e. {@link #init}
-     * and {@link #close} can be called multiple times, possibly in parallel.
+     * Returns {@code true} if this instance can be used in different vertices
```

Comment:
different vertices? AFAIR PMSes are not shared between vertices, each Vertex has its' own PMS

Reply:
techincally PMS can be used for many vertices, but that should be rare